Lib/test/test_sys.py

```diff
@@ -1076,14 +1076,21 @@ class B:
                 class X(Exception):
                     pass
 
```

Comment:
I missed the review of your nice enhancement (qualname). Would you mind to add a comment mentioning bpo-41031 and explainaing that the test purpose is to ensure that an exception contains the qualified name of an exception class, and not just its name?